jnigen/test/test_util/test_util.dart

```diff
@@ -88,9 +88,10 @@ void comparePaths(String path1, String path2) {
   if (diffProc.exitCode != 0) {
     final originalDiff = diffProc.stdout;
     log.warning(
-        "Paths $path1 and $path2 differ, comparing by ignoring space change");
-    final fallbackDiffProc = Process.runSync(
-        "git", [...diffCommand, '--ignore-space-change', path1, path2]);
+        "Paths $path1 and $path2 differ, Running dart format on $path1.");
+    Process.runSync('dart', ['format', path1]);
+    final fallbackDiffProc =
+        Process.runSync("git", [...diffCommand, path1, path2]);
     if (fallbackDiffProc.exitCode != 0) {
       stderr.writeln(originalDiff);
```

Comment:
Wouldn't you want to print `fallbackDiffProc.stdout`/`fallbackDiffProc.stderr` here?

Reply:
`dart format` is an exceptional thing that happens only when stable != beta. When even after formatting the diff produces a non-zero result, I show whatever happened originally before formatting.

Reply:
Shouldn't the fallback diff be more useful? Otherwise you get all the things that are not actually diffs.

Reply:
I understand your point. My logic is: if beta fails, stable will surely fail too, so we get the good diff there. `dart format` is only a hack to fix the CI in our current situation. And if this is no longer the case, we're falling back to the original way. (As if we never had the `dart format`).